CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `date` and `modified` degree program REST API properties.
+- `fau/v1/degree-program/index` REST API endpoint.
+
```

Comment:
TBH, I'm not sure if we want these changelog entries here in the embed plugin. Since we have them already in the main plugin, let's remove them, unless you oppose.
```suggestion
```

Reply:
The thing is, the consuming websites have REST API as well (while it is used internally only I guess). But I don't mind to remove.